### DISCUSSION TITLE
Wiki not accessible through URL after digitalocean installation

### DISCUSSION THREAD

**Discussion by marcmaurer99** [requarks/wiki#4569] [Help / Questions] [answered]
Hi there,

I used the digitalocean installation to install my instance of wiki.js instance. I can access it as admin no problem but when I want I to access it via the general URL 

http://www.wiki.maurer-group.com/

I get this error message;

Hmmm… can't reach this pageCheck if there is a typo in wiki.maurer-group.com.
Search Bing for wiki maurer-group
If spelling is correct, try running Windows Network Diagnostics.
DNS_PROBE_FINISHED_NXDOMAIN

Does it take some time until this works after the digitalocean installation or is something wrong?

Thanks,
Marc

**Comment by NGPixel** [comment, marked as answer]
This is not an issue with Wiki.js. Make sure your CNAME / A DNS record is pointing to the proper IP.
